src/Repository/SnowCategoryRepository.php

```diff
@@ -19,6 +19,23 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, SnowCategory::class);
     }
 
+    /**
+     * Method findOneByName.
+     *
+     * @param $value $value returns the name of the category
+     *
+     * @return SnowCategory
+     */
+    public function findOneByName($value): ?SnowCategory
```

Comment:
il faut typer la valeur d'entrée de ta fonction et modifier le commentaire en conséquence.
```suggestion
    public function findOneByName(string $value): ?SnowCategory
```

Reply:
done